_posts/batlopez/2016-02-03-NatsChap5Exerc.md

```diff
@@ -0,0 +1,11 @@
+--- 
+layout: post
+author: batlopez
+title: "Nat's Chapter 5 Exercises!"
+---
+Exercise 1: 
+<iframe src="https://trinket.io/embed/python/61d804b72e" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
Nicely done.  ONe thing I'd suggest is breaking out the stuff that happens at the end outside of the loop: https://trinket.io/python/dfef1a65e4